### DISCUSSION TITLE
Is there any permission in TownyAdvanced that allows controlling the maximum number of residents in a town, or is there an additional plugin that can provide this functionality?

### DISCUSSION THREAD

**Discussion by xiaoluofa** [TownyAdvanced/Towny#7669] [Q&A] [answered]
Is there any permission in TownyAdvanced that allows controlling the maximum number of residents in a town, or is there an additional plugin that can provide this functionality?

**Comment by LlmDl** [comment, marked as answer]
In the config you have the option:

```
  # The maximum number of residents that can be joined to a town. Setting to 0 disables this feature.
  max_residents_per_town: '0'
```

**Reply by xiaoluofa** [reply]
I know if there are any other ways for some people to get more.

**Comment by LlmDl** [comment]
There is not other ways. If you wanted to commission a custom plugin would
be able to do it. See the discord for details.
